### PR TITLE
Finished 2 of 5

### PR DESCRIPTION
**Pull request by Bkliger**
Quit before formatting the final elapsed time in the stopwatch. Oh well..........

### CODE REVIEW

**Comment by andreaainara** [comment]
This is still a bit difficult for me... I was able to make sense of and eventually get close enough to the code for the solutions for some of the problems, but for "change", I was unable to figure it out

File: ready/js/app.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/*
+  What needs to change so that the page says "Go!"?
+*/
+
+$(document).ready(function() {
+
```
Comment (Nick-Brennan) [inline ready/js/app.js:6]: It doesn't look like you set the h1's text to "Go!!!".  Did you miss a step?

**Comment by Nick-Brennan** [comment]
I don't see your work.  Perhaps you made the pull request without first adding, committing and pushing your changes?

**Comment by andreaainara** [comment]
ahh, that's entirely possible... I need to go check

**Pull request closed without merging**